tasks/actorstate/power_test.go

```diff
@@ -0,0 +1,131 @@
+package actorstate
+
+import (
+	"context"
+	"testing"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/lotus/chain/actors/builtin/power"
+	"github.com/filecoin-project/lotus/chain/types"
+	powermodel "github.com/filecoin-project/sentinel-visor/model/actors/power"
+	sa0builtin "github.com/filecoin-project/specs-actors/actors/builtin"
+	sa0smoothing "github.com/filecoin-project/specs-actors/actors/util/smoothing"
+	sa2builtin "github.com/filecoin-project/specs-actors/v2/actors/builtin"
+	sa2smoothing "github.com/filecoin-project/specs-actors/v2/actors/util/smoothing"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPowerExtractV0(t *testing.T) {
+	ctx := context.Background()
+
+	mapi := NewMockAPI()
+
+	state, err := mapi.newEmptyPowerStateV0()
+	require.NoError(t, err)
+
+	state.TotalRawBytePower = abi.NewStoragePower(1000)
+	state.TotalBytesCommitted = abi.NewStoragePower(0)
+	state.TotalQualityAdjPower = abi.NewStoragePower(0)
+	state.TotalQABytesCommitted = abi.NewStoragePower(0)
+	state.TotalPledgeCollateral = abi.NewTokenAmount(0)
+	state.ThisEpochRawBytePower = abi.NewStoragePower(0)
+	state.ThisEpochQualityAdjPower = abi.NewStoragePower(0)
+	state.ThisEpochPledgeCollateral = abi.NewTokenAmount(0)
```

Comment:
Going to add some values here to make sure we don't get false positives due to zero values conveniently matching our test data.